integration-cli/docker_cli_network_unix_test.go

```diff
@@ -201,7 +201,8 @@ func isNwPresent(c *check.C, name string) bool {
 	out, _ := dockerCmd(c, "network", "ls")
 	lines := strings.Split(out, "\n")
 	for i := 1; i < len(lines)-1; i++ {
```

Comment:
Can we clean up this for loop?

```go
for _, line := range lines {
    if strings.Fields(line)[1] == name {
        return true
    }
}
```

Might also need a trim space on `out` before the split.